k2/python/tests/fsa_test.py

```diff
@@ -227,7 +285,7 @@ def test_transducer_from_str(self):
             5 0  1  50  -8.2
             6
         '''
-        fsa = k2.Fsa.from_str(_remove_leading_spaces(s))
+        fsa = k2.Fsa.from_str(_remove_leading_spaces(s), num_aux_labels=1)
```

Comment:
The test code covers only the case for `num_aux_labels=1`.

I am worrying that most of the code is actually not tested.